app/components/gh-posts-list-item.hbs

```diff
@@ -51,7 +51,7 @@
         {{#if (and this.feature.emailAnalytics (eq @post.displayName "post"))}}
             <LinkTo @route="editor.edit" @models={{array @post.displayName @post.id}} class="permalink gh-list-data gh-post-list-recipients">
                 <div class="flex fw4">
-                    {{#if (or @post.email @post.willEmail)}}
```

Comment:
Removed this because I don't think this made sense because the next line specifically checks for email, and a scheduled, not sent post, will never have email counts.

Reply:
👍 We can remove the conditional completely, the following `{{#if (eq @post.email.status "submitted")}}` is all that's needed

Reply:
Removed 🔥